src/main/java/de/retest/recheck/cli/subcommands/Ignore.java

```diff
@@ -73,7 +74,7 @@ public void run() {
 				}
 				TestReportUtil.print( report, testReport );
 				loadRecheckIgnore();
-				if ( !ignoreDifferences( report ) ) {
+				if ( !ignoreDifferences( new TestReportFilter( ignoreApplier ).filter( report ) ) ) {
```

Comment:
Do you want to extract that `TestReport` into a variable `filteredTestReport` or `notIgnoredDifferencesReport` so that it is more clear that the actual report contains only the non ignored differences and elements?